src/content.ts

```diff
@@ -982,6 +985,25 @@ function previewTime(time: number, unpause = true) {
     }
 }
 
+//send telemetry and count skip
+function sendTelemetryAndCount(skippingSegments: SponsorTime[], secondsSkipped: number, fullSkip: boolean) {
+    let counted = false
+    for (const segment of skippingSegments) {
+        const index = sponsorTimes.indexOf(segment);
+        if (index !== -1 && !sponsorSkipped[index]) {
+            if (Config.config.trackViewCount) {
```

Comment:
`if (Config.config.trackViewCount) return` at the top of the function is probably a tiny bit more efficient.